spec/ndx-microscopy.extensions.yaml

```diff
@@ -325,3 +410,79 @@ groups:
       doc: Link to VolumetricImagingSpace object containing metadata about the region of physical space this imaging data
         was recorded from.
       target_type: VolumetricImagingSpace
+
+
+  - neurodata_type_def: VariableDepthMultiChannelMicroscopyVolume
```

Comment:
Also adding a variable-depth static volume

TODO: reduce this block using inheritance